README.md

```diff
@@ -308,6 +308,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 - [GNUnet](https://gnunet.org/) - Free software framework for decentralized, peer-to-peer networking. ([Source Code](https://gnunet.org/git/)) `GPL-3.0` `C`
 - [Gotify](https://gotify.net/) - Self-hosted notification server with Android and CLI clients, similar to PushBullet. ([Source Code](https://github.com/gotify/server), [Clients](https://github.com/gotify/android)) `MIT` `Go`
 - [Hawkpost](https://hawkpost.co) - HawkPost is a web app that lets you create unique links that you can share with a person that desires to send you important information but doesn't know how to encrypt it. The message is encrypted in their browser and sent to your email address. ([Source Code](https://github.com/whitesmith/hawkpost)) `MIT` `Python`
+- [IndiCom](https://github.com/balzack/databag) - Federated messaging client for the web, supporting text, photos and video. ([Demo](https://databag.coredb.org/#/create)) `Apache-2.0` `Go`
```

Comment:
The naming is a bit confusing. The repo says the linked project is called "databag" and "includes IndiCom"

Reply:
The backend (databag) is an implementation of an API that can support other types of services, for example contact apps, or photo sharing apps. Currently I have implemented a messaging service "IndiCom", but I plan to write other services in time. Would you prefer if I rename the entry as "Databag"?

Reply:
In that case I think it would be best to split the repository. One repo for your backend and then one repo for each of the example apps (like Indicom).

Reply:
I see your point, and I will split them up when I start on a second service, but for now it is helpful to me that they be in the same repository. I think I jumped the gun naming the service differently. How about I rename the service to 'Databag' for now, for clarity?

Reply:
Yes, i think that would make it clearer. But just to have it said, this is just the opinion of a stranger on the internet. It does not mean that your PR will be easier merged into this project.

Reply:
I understand, but you have a valid point. I will make this change in the Databag project, at this point it's just a name in the web app and the project readme.

Reply:
I have made the change to the Databag project and updated the merge request. Thanks for the feedback.